packages/slate-editor/src/modules/decorations/context.ts

```diff
@@ -0,0 +1,15 @@
+import type { Decorate } from '@prezly/slate-commons';
+import { noop } from '@technically/lodash';
+import { createContext } from 'react';
+
+type Detach = () => void;
+
+export type DecorationsConnector = {
+    attach(decorate: Decorate): Detach;
+};
+
+export const decorationsContext = createContext<DecorationsConnector>({
```

Comment:
I believe the naming convention for context objects is PascalCase, so that it doesn't look weird in JSX 🤔

Reply:
56852fe352041753230e071214a0a075dc6b3bd1